tools/on_device_tests_gateway_client.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""gRPC On-device Tests Gateway client."""
+
+import argparse
+import logging
+import sys
+
+import grpc
+
+import on_device_tests_gateway_pb2
+import on_device_tests_gateway_pb2_grpc
+
+
+# All tests On-Device tests support
+_TEST_TYPES = [
+    'unit_test',
+    'black_box_test',
+    'evergreen_test',
+    'performance_test',
+    'endurance_test',
+    'integration_test',
+    'upload_archive',
+]
+
+# All platforms On-Device tests support
+_PLATFORM_TYPES = [
```

Comment:
I don't see 'evergreen-arm-hardfp' on this list, see an example in Buildbot: https://cobalt-buildbot.corp.goog/builders/raspi-2_evergreen-arm-hardfp_evergreen_tests/builds/276/steps/Trigger%20Mobile%20Harness%20evergreen_test%20Config%20qa/logs/stdio

Reply:
Resolved